README.md

```diff
@@ -1,122 +1,241 @@
+A tool to synchronize an RRDP and/or Rsync server with a "hidden" remote 
+[RFC 8182](https://tools.ietf.org/html/rfc8182) RRDP publication point.
 
-## TL;DR
+## Installation
+
+### Install using Pre-built Packages
+
+Assuming you have a machine running a recent Debian or Ubuntu distribution, you 
+can install krill-sync from our [software package repository](https://packages.nlnetlabs.nl).
+To use this repository, add the line below that corresponds to your operating system to 
+your `/etc/apt/sources.list` or `/etc/apt/sources.list.d/`
+
+```bash
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/debian/ stretch main
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/debian/ buster main
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/ubuntu/ xenial main
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/ubuntu/ bionic main
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/ubuntu/ focal main
+```
+Then run the following commands.
+
+```bash
+sudo apt update && apt-get install -y gnupg2
+wget -qO- https://packages.nlnetlabs.nl/aptkey.asc | sudo apt-key add -
+sudo apt update
+```
+
+You can then install, enable and start Krill by running
+
+```bash
+sudo apt install krill-sync
+```
+
+
+### Build with Cargo
 
 For Ubuntu 20.04 with Rust 1.51.0:
 
 ```
 $ apt update && apt install -y build-essential curl libssl-dev openssl pkg-config
 $ curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 $ source $HOME/.cargo/env
-$ cargo install --git https://github.com/NLnetLabs/krill-sync.git --tag v0.1.2 --locked
-$ krill-sync https://<hidden fqdn of RRDP publication point>/rrdp/notification.xml -v
-...
+$ cargo install --git https://github.com/NLnetLabs/krill-sync.git --tag v0.2.0 --locked
 ```
 
-**Tada!** Rsync and RRDP directories have now been created locally which can be used to serve the content produced by the remote "hidden" RRDP origin server via the RRDP and Rsync protocols. Note: this tool does not serve them for you, for that you need NGINX and Rsyncd for example.
-
 ## Introduction
 
-This tool is intended to enable public facing RRDP and Rsync servers to serve consistent and fresh content obtained from a "hidden" RRDP master server, e.g. that runs a Krill RPKI Certificate Authority and RPKI publication repository. In this context "hidden" means that the server is not reachable by the public name of the repository but creates content that points to the public name. Krill Sync is smart enough to know that it shouldn't blindly follow FQDNs in the RRDP content to download snapshots and deltas but instead goes to the specified "hidden" FQDN each time instead.
+This tool is intended to enable public facing RRDP and Rsync servers to serve consistent
+and fresh content obtained from a "hidden" RRDP master server, e.g. that runs a Krill 
```

Comment:
Use "origin" or "source" rather than "master"?

Reply:
Also, this is no longer Krill specific according to the updated opening lines of the README so a reference to Krill doesn't seem right here.